Bottom: d8381b2e90d165e5c2880b56f361eab0f237f8e4
Top:    b1d6dda42bad2b028b15b43f5f8226f035889089
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-13 22:48:27 +0000

Refresh of fix-some-warnings-due-to

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index e526875..c46440f 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1298,7 +1298,7 @@ Load (int argc,
   if (name)
     {
       if (Settings.verbose)
-      	fprintf (stderr, "%s:  Calling LoadFrom(%s, %s)\n", G_STRFUNC,
+	fprintf (stderr, "%s:  Calling LoadFrom(%s, %s)\n", G_STRFUNC,
 		 function, name);
       hid_actionl ("LoadFrom", function, name, NULL);
       g_free (name);
